pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -410,13 +401,11 @@ func (dm *DockerManager) inspectContainer(id string, podName, podNamespace strin
 		// Container that are running, restarting and paused
 		status.State = kubecontainer.ContainerStateRunning
 		status.StartedAt = startedAt
-		if containerName == PodInfraContainerName {
```

Comment:
My thinking was that `inspectContainer` should return the IP address for any given container. We then determine the pod IP in `GetPodStatus`, using the `containerProvidesPodIP` func to determine which container's IP we should use. On the Linux side, we want the infra container's IP. On the windows side, we don't want the infra container's IP.